Postman/dex.postman_collection.json

```diff
@@ -1640,6 +1640,20 @@
 								}
 							]
 						},
+						{
+							"name": "UserProjects",
+							"item": [
+								{
+									"name": "UserProject-GetProject-Self-Administrator",
+									"request": {
+										"method": "GET",
+										"header": [],
+										"url": null
```

Comment:
As we spoke about on Teams, all looks good except this url probably caused the tests to fail.